### DISCUSSION TITLE
60%-40% split

### DISCUSSION THREAD

**Discussion by koenigdavidmj** [rxhanson/Rectangle#735] [Ideas]
I'm thinking of a modification to the corner options (Top Left, Bottom Right, etc). We would add a setting consisting of a split percentage. For example, if it was 60%, then Top Left would make a window 60% of the width of the screen, and Top Right would be 40% of the width.

We'd probably also want to support a vertical percentage, and maybe support this for sixths/ninths too.

If I could find the time, would such a PR be welcome?

**Comment by rxhanson** [comment]
Thanks for looking to contribute! Sure, I am open to merging a PR for this. 

At first cut, I prefer not to add anything to the UI because it is kept purposefully slim, and it's easier to build out features quickly when they are configured via terminal command. It sounds like this would either be alternate behavior on the current corner actions, or a new set of actions. 

If you want to take the route of adding a new set of actions, reference the PR for ninths https://github.com/rxhanson/Rectangle/pull/702 (the most recent one done for terminal configured actions).

**Comment by 972creative** [comment]
Hi there. I'm trying out Rectangle Pro right now. Was any progress made on this 60-40 split? It's exactly what I'm trying to do.

**Reply by rxhanson** [reply]
I'm still fine with merging a PR for something like this into Rectangle as terminal command configurations.

In Rectangle Pro, there's a custom size & position builder that you could use to create a 60 40 split if desired. (Note that you can use fraction or decimal values less than 1 for a percentage). Let me know if you have trouble building it out there.
